src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/SvDiscoverFromLocalAssemblyContigAlignmentsSpark.java

```diff
@@ -34,21 +34,50 @@
 import static org.broadinstitute.hellbender.tools.spark.sv.discovery.prototype.AssemblyContigAlignmentSignatureClassifier.RawTypes;
 
 /**
- * This tool takes a SAM file containing alignments of single-ended long read
- * (be it long read sequencing, or contigs assembled from standard Illumina short reads--currently the primary target),
- * searches for reads with--potentially complicated--split alignments indicating the presence of SV breakpoints.
- * In case of complex structural variations (cxSV), also outputs a custom file format containing how affected
- * reference segments are rearranged on the provided sample where the long reads are from.
+ * Parse aligned contigs and call complex structural variants.
```

Comment:
This is my bad: not updating the documentation as the code evolves.

It now actually handles all types of SV's based solely on scanning the alignment signature of assembly contigs.

So it could be read as 
"Parse aligned contigs and emit VCF records for structural variants based on contigs' alignment signatures."